frontend/src/metabase/query_builder/components/NativeQueryEditor/NativeQueryEditor.tsx

```diff
@@ -12,15 +12,9 @@ import "ace/ace";
 import "ace/ext-language_tools";
 import "ace/ext-searchbox";
 import "ace/mode-sql";
-import "ace/mode-mysql";
-import "ace/mode-pgsql";
-import "ace/mode-sqlserver";
```

Comment:
We should also remove related mocks:
https://github.com/metabase/metabase/blob/cd852d225526178a034cc79dfce38b8995d2e415/frontend/test/__support__/mocks.js#L29-L31

https://github.com/metabase/metabase/blob/cd852d225526178a034cc79dfce38b8995d2e415/frontend/test/__support__/mocks.js#L34-L36